src/components/BooksForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import categories from '../variables/Categories';
+
+const BookForm = () => (
+  <form>
```

Comment:
- [ ] As it said in the instructions, we need to Separate components in Presentational Components and Container Componentes. This approach can make your app easier to understand and allow you to more easily reuse components. In your components directory (presentational components) you have components that are not connected to the Redux store, and on your containers directory, you have components that are connected to the Redux store. This is the common approach followed when using Redux with React. Check out [here](https://redux.js.org/basics/usage-with-react#presentational-and-container-components). Right now BookForm does not connect with Redux yet, but we will do that in a future milestone. We need to move BookForm from `components` directory to the `containers` directory.

Reply:
Thanks Sergio for taking the time to explain these concepts. :)